### ISSUE TITLE
Add line number selection

### ISSUE THREAD

**Issue by jackyzha0** [jackyzha0/ctrl-v#69]

**Comment by Siddhant-K-code** [comment]
Do you want something similar to this? :

![image](https://user-images.githubusercontent.com/55068936/114393798-11cf0600-9bb8-11eb-8bef-9720d65334ad.png)

**Comment by jackyzha0** [comment]
> Do you want something similar to this? :
> 
> ![image](https://user-images.githubusercontent.com/55068936/114393798-11cf0600-9bb8-11eb-8bef-9720d65334ad.png)

ctrl-v already has something like this in the 'view paste' mode but I wanted to add the ability to select a range of line numbers to highlight and copy a permalink to (similar to GitHub's functionality)

Will update the issue with more info in a bit

**Comment by jackyzha0** [comment]
@Siddhant-K-code updated, let me know if this is still something you'd like to work on!

**Comment by Siddhant-K-code** [comment]
> @Siddhant-K-code updated, let me know if this is still something you'd like to work on!

ok, I will look into it, can you please check your mail once :)

**Comment by Siddhant-K-code** [comment]
> Example URL would be https://ctrl-v.app/2uKMJEu#L1-L14

Line Numbering is Broken? :

![image](https://user-images.githubusercontent.com/55068936/114496936-9cf2df00-9c3e-11eb-8bcb-05c324ddef96.png)

No Line Number is not showing after *`64`*

**Comment by jackyzha0** [comment]
> ctrl-v.app/2uKMJEu#L1-L14

hm that's really funky, what's your screen size? 
![image](https://user-images.githubusercontent.com/23178940/114497235-87e26580-9bd6-11eb-9369-809c2dffa82d.png)

**Comment by Siddhant-K-code** [comment]
Idk, what is the issue, it is looking the same on multiple monitors and even after increasing/decreasing screen resolution.

**Comment by jackyzha0** [comment]
> Idk, what is the issue, it is looking the same on multiple monitors and even after increasing/decreasing screen resolution.

ah, just realized this is a CSS change I made to line-breaking in the `pre`, `code`, and `textarea` tags oof, do you mind filing this as a separate issue?

**Comment by Siddhant-K-code** [comment]
no, we can cover this under it also there are a lot of CSS changes required like, the Watermark at the left bottom is not responsive!

**Comment by Siddhant-K-code** [comment]
Or, we can create a new issue for fixing all CSS! till then can you please tell color palette for #63

**Comment by jackyzha0** [comment]
> Or, we can create a new issue for fixing all CSS! till then can you please tell color palette for #63

this would be preferred! having an issue for all CSS related ones would be nice. Let me get you the palette

**Comment by Siddhant-K-code** [comment]
created #79 for all CSS stuff
